packages/fxa-auth-server/docs/swagger/auth-server-api.ts

```diff
@@ -43,13 +41,13 @@ export const AUTH_SERVER_API_DESCRIPTION = {
 
     For example:
 
-    > \`{\`<br/>
-    >   \`"code": 400,\`  // Matches the HTTP status code<br/>
-    >   \`"errno": 107,\` // Stable application-level error number<br/>
-    >   \`"error": "Bad Request",\` // String description of the error type<br/>
-    >   \`"message": "Invalid parameter in request body",\` // Specific error message<br/>
-    >   \`"info": "https://docs.dev.lcip.og/errors/1234"\`  // Link to more information<br/>
-    > \`}\`
+        {
+          "code": 400,  // Matches the HTTP status code
+          "errno": 107, // Stable application-level error number
+          "error": "Bad Request", // String description of the error type
+          "message": "Invalid parameter in request body", // Specific error message
+          "info": "https://docs.dev.lcip.og/errors/1234"  // Link to more information
+        }
 
```

Comment:
Q: I assume the indenting trick is to appease the Markdown code formatting gods? https://daringfireball.net/projects/markdown/syntax#precode
> _"To produce a code block in Markdown, simply indent every line of the block by at least 4 spaces or 1 tab. For example, given this input:"_

Does code fencing not work here? Curious if a fence would give you built-in code styling as well. (and double assuming you'd need to escape each of the backticks due to **dedent**):
````
\`\`\`js
    {
      "code": 400,  // Matches the HTTP status code
      "errno": 107, // Stable application-level error number
      "error": "Bad Request", // String description of the error type
      "message": "Invalid parameter in request body", // Specific error message
      "info": "https://docs.dev.lcip.og/errors/1234"  // Link to more information
    }
\`\`\`
````